File: autofin/src/main/java/model/vehicle_details/vehicle_category/stock_details/StockResponse.java
package model.vehicle_details.vehicle_category.stock_details;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class StockResponse {

    @SerializedName("status")
    @Expose
    private Boolean status;
    @SerializedName("message")
    @Expose
    private String message;
    @SerializedName("statusCode")
    @Expose
    private String statusCode;
    @SerializedName("data")
    @Expose
    private StockResponse data;

    public Boolean getStatus() {
        return status;
    }

    public void setStatus(Boolean status) {
        this.status = status;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public String getStatusCode() {
        return statusCode;
    }

    public void setStatusCode(String statusCode) {
        this.statusCode = statusCode;
    }

    public StockResponse getData() {
        return data;
    }

    public void setData(StockResponse data) {
        this.data = data;
    }

}

File: autofin/src/main/java/v2/model/response/StockResponse.kt
package v2.model.response

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
data class StockResponse
(
    @SerializedName("status") var status : Boolean=false,
    @SerializedName("message") var message : String="",
    @SerializedName("statusCode") var statusCode : String="",
    @SerializedName("data") var data : StockDetails? = StockDetails()

    ): Parcelable